.changesets/fix_bryn_datadog_trace_id.md

```diff
@@ -0,0 +1,18 @@
+### Attach `dd.trace_id` to json formatted log messages ([PR #4764](https://github.com/apollographql/router/pull/4764))
+
+To enable correlation between DataDog tracing and logs, `dd.trace_id` must appear as a span attribute and also on the root of each `json` formatted log message.
+If users configure their router.yaml to include `dd.trace_id`:
```

Comment:
```suggestion
Once you configure the `dd.trace_id` attribute in router.yaml, it will automatically be extracted from the root span and attached to the logs:
```